include/common.php

```diff
@@ -1829,13 +1829,11 @@ function get_labels_xml_from_query_results($qry)
     return $xml;
 }
 
-function generate_web_api_key()
+function generate_password($length)
```

Comment:
Tempting, but it doesn't quite seem like a good fit to me.

Using the raw output from `random_bytes` can generate exotic characters that potentially break the user registration process.  (Plus some users might have a hard time entering characters like ݩ)

Passing it through `bin2hex` like we do in the example you linked reduces the size of our character set from 62 to 16.